docs/source/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Apollo Link
+sidebar_title: Introduction
+description: Apollo Link is a standard interface for modifying control flow of GraphQL requests and fetching GraphQL results.
+---
+
+This is the official guide for using Apollo Links in your Apollo app using the [apollo-link](https://github.com/apollographql/apollo-link) package. Apollo Link is a simple yet powerful way to describe how you want to execute a GraphQL operation, and what you want to do with the results. It is a convenient way to customize Apollo Client to the needs of your application while providing a small API to learn.
```

Comment:
Hmm - as a branding thing, should we say "Apollo Links" or "Apollo Link"?

Reply:
I think `Apollo Link` since that is also the name of the interface